Spanner/tests/Unit/ResultTest.php

```diff
@@ -317,7 +317,7 @@ public function testSnapshot()
         $fixture = $this->getStreamingDataFixture()['tests'][1];
         $result = $this->getResultClass($fixture['chunks']);
 
-        $this->assertNull($result->snapshot());
+        $this->assertInstanceOf(Snapshot::class, $result->snapshot());
```

Comment:
Same as the assertion after `next()` call
```suggestion
```

Reply:
Removed the `next()` and assertion after `next()` as the `transaction`/`snapshot` gets created in the constructor of `Result` itself.